src/metadata/trackmetadatataglib.cpp

```diff
@@ -18,6 +18,12 @@
 #define TAGLIB_HAS_TAG_CHECK \
     (TAGLIB_MAJOR_VERSION > 1) || ((TAGLIB_MAJOR_VERSION == 1) && (TAGLIB_MINOR_VERSION >= 9))
 
+#ifdef _WIN32
+#define TAGLIB_FILENAME_FROM_QSTRING(fileName) (fileName).toStdWString().constData();
```

Comment:
I will push 2 PRs (master + 1.12): We have to use c_str() instead of data() 
for conversion from std::wstring!

On 10/13/2015 09:00 AM, Daniel Schürmann wrote:
>
> In src/metadata/trackmetadatataglib.cpp 
> <https://github.com/mixxxdj/mixxx/pull/734#discussion_r41832547>:
>
> > @@ -18,6 +18,12 @@
> >  #define TAGLIB_HAS_TAG_CHECK \
> >      (TAGLIB_MAJOR_VERSION > 1) || ((TAGLIB_MAJOR_VERSION == 1) && (TAGLIB_MINOR_VERSION >= 9))
> >
> > +#ifdef _WIN32
> > +#define TAGLIB_FILENAME_FROM_QSTRING(fileName) (fileName).toStdWString().constData();
>
> I have just noticed that there is a typo.
> #define TAGLIB_FILENAME_FROM_QSTRING(fileName) 
> (fileName).toStdWString().const();
>
> We should fix it together with:
> #731 (comment) 
> <https://github.com/mixxxdj/mixxx/pull/731#issuecomment-147366056>
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/mixxxdj/mixxx/pull/734/files#r41832547>.
>